packages/twenty-front/src/modules/auth/sign-in-up/components/SignInUpForm.tsx

```diff
@@ -27,6 +27,11 @@ import { TextInput } from '@/ui/input/components/TextInput';
 import { ActionLink } from '@/ui/navigation/link/components/ActionLink';
 import { isDefined } from '~/utils/isDefined';
 
+const validateEmail = (email: string) => {
```

Comment:
We already have email validation here in the app, no need to introduce a new function :)
https://github.com/twentyhq/twenty/pull/7245/files#diff-a459c86227f1320a782da50d5467cddf0f21787f5bb1747396b643f5ea7c7c1aR35

Reply:
Thanks, Will add it :)